docs/swagger.yaml

```
@@ -557,6 +557,31 @@ paths:
       summary: Get logged-in user data
       tags:
       - user
+  /user/oib/{oib}:
+    get:
+      description: get a user with oib
+      parameters:
+      - description: user oib
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)